packages/blockchain-wallet-v4-frontend/src/data/modules/profile/selectors.ts

```diff
@@ -166,3 +167,11 @@ export const isExchangeRelinkRequired = (state): RemoteDataType<string, boolean
   lift((user: UserDataType) => {
     return not(isNil(prop('settings', user))) && length(getRemainingCoins(state))
   })(getUserData(state))
+
+export const isFlowInRiskSettings = (state: RootState, flowName: string): boolean => {
+  const userRiskSettings = getUserRiskSettings(state).getOrElse({ flows: [] } as UserRiskSettings)
```

Comment:
```suggestion
  const userRiskSettings: UserRiskSettings = getUserRiskSettings(state).getOrElse({ flows: [] })
```

I would avoid type casting if possible.